ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/ActionConfigureComponentJms.java

```diff
@@ -0,0 +1,47 @@
+package io.syndesis.qe.pages.integrations.edit;
+
+import static com.codeborne.selenide.Condition.visible;
+
+import java.lang.reflect.Field;
+
+import org.openqa.selenium.By;
+
+import com.codeborne.selenide.SelenideElement;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public abstract class ActionConfigureComponentJms extends ActionConfigureComponent {
+
+	abstract Class getInputClass();
+
+	abstract Class getSelectClass();
+
+	public SelenideElement checkAndGetFieldType(String elementId) throws IllegalArgumentException, IllegalAccessException {
```

Comment:
Here it's. Those exception shouldn't be needed.

Reply:
@dsimansk Solved